README.md

```diff
@@ -1 +1,102 @@
-# aar-operator
\ No newline at end of file
+# Anbox Application Registry (AAR) Charmed Operator
+
+## Description
+
+### Anbox Cloud
+
+Anbox Cloud offers a software stack that runs Android applications in any cloud enabling high-performance
+streaming of graphics to desktop and mobile client devices.
+
+At its heart, it uses lightweight container technology instead of full virtual machines to achieve higher
+density and better performance per host while ensuring security and isolation of each container. Depending
+on the target platform, payload, and desired application performance (e.g. frame rate), more than
+100 containers can be run on a single machine.
+
+For containerization of Android, Anbox Cloud uses the well established and secure container hypervisor
+LXD. LXD is secure by design, scales to a large number of containers and provides advanced resource
+management for hosted containers.
+
```

Comment:
For the entire introduction, I would recommend to mimic the changes in the [VM support branch](https://github.com/anbox-cloud/discourse-docs/blob/1a233aacd3e14d30947a168252387b4702b2482a/exp/anbox-cloud.md)